### DISCUSSION TITLE
Billboard.js 3.5.1 with angular 6.0 gives ERROR TypeError: t.hasType is not a function when ng serve --prod

### DISCUSSION THREAD

**Discussion by flocheke7** [naver/billboard.js#2883] [Q&A]
I try to use Billboard.js 3.5.1 with an angular6.0 application
 when i run with ng serve , application works fine. 
when I run with ng serve --prod, application gives the following error when application gerenates chart (bb.generate)

main.19bd664034f5a0e96060.js:formatted:7402 ERROR TypeError: t.hasType is not a function at main.19bd664034f5a0e96060.js:formatted:66053:39 at e.init (main.19bd664034f5a0e96060.js:formatted:66067:22) at new e (main.19bd664034f5a0e96060.js:formatted:66471:23) at Object.generate (main.19bd664034f5a0e96060.js:formatted:69686:29) at e.ngAfterViewInit (main.19bd664034f5a0e96060.js:formatted:139959:35) at fa (main.19bd664034f5a0e96060.js:formatted:11674:55) at ha (main.19bd664034f5a0e96060.js:formatted:11662:36) at da (main.19bd664034f5a0e96060.js:formatted:11655:60) at Pa

can i use billboard 3.5.1 with angular 6 ?

**Comment by netil** [comment]
duplicated issue:
- https://github.com/naver/billboard.js/issues/2885
